docs/01-app/03-api-reference/05-config/02-typescript.mdx

```diff
@@ -131,6 +131,15 @@ export default nextConfig
 
 Next.js will generate a link definition in `.next/types` that contains information about all existing routes in your application, which TypeScript can then use to provide feedback in your editor about invalid links.
 
+> **Good to know**: If you set up your project without `create-next-app`, ensure the generated Next.js types are included by adding `.next/types/**/*.ts` to the `include` array in your `tsconfig.json`:
+>
+> ```json filename="tsconfig.json"
+> {
+>   "include": ["next-env.d.ts", ".next/types/**/*.ts", "**/*.ts", "**/*.tsx"],
+>   "exclude": ["node_modules"]
+> }
+> ```
```

Comment:
This works, but it looks a bit odd, frame within a frame - maybe it is ok to be a separate snippet block


<img width="776" height="474" alt="Screenshot 2025-08-26 at 00 16 54" src="https://github.com/user-attachments/assets/2bc4914e-7b65-41f8-afa7-3d15e76a2c16" />